docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<p>where are we using torch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I used a clean environment, and asked me to install it to run something in the pipeline</p>

Reply:
@gabrielmbmb is this intended? I would assume we would not need this when just working with InferenceEndpoints.